### PR TITLE
Feature/ecommerce/user

### PR DESCRIPTION
**Pull request by eefborromeo**
Creates "Add to Cart" feature. User can add to cart and remove item from cart. This is related to User Story #5.

### CODE REVIEW

File: app/models/cart_item.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  belongs_to :user
+  belongs_to :product
+
+  before_create :check_quantity
+
+  private
+
+  def check_quantity
+    specific_product = Product.find(self.product_id)
+    puts specific_product
```
Comment (lkpallarca) [inline app/models/cart_item.rb]: you can remove the puts

**Review by lkpallarca — APPROVED** [review]
looks good 💯

File: app/models/cart_item.rb (hunk 1 above)
Comment (eefborromeo) [reply app/models/cart_item.rb]: I created an additional commit to remove the puts

https://github.com/eefborromeo/private-dermatology/pull/22/commits/4e18161324461113bc2aab818c903d902adb52d7

**Comment by jarayumang** [comment]
This is great!

**Pull request merged**